v2/concordium/protocol-level-tokens.proto

```diff
@@ -153,13 +140,13 @@ message TokenGovernanceEvent {
 // Token events originating from token governance transactions.
 message TokenGovernanceEffect {
   // Events emitted by the token.
-  repeated TokenGovernanceEvent events = 1;
+  repeated TokenEvent events = 1;
 }
 
 // Token events originating from token holder transactions.
 message TokenHolderEffect {
   // Events emitted by the token.
-  repeated TokenHolderEvent events = 1;
+  repeated TokenEvent events = 1;
 }
```

Comment:
Since these types are functionally identical, they should be merged.

Reply:
Good point